halo2-base/src/lib.rs

```diff
@@ -133,6 +133,12 @@ pub struct AssignedValue<F: crate::ff::Field> {
     pub cell: Option<ContextCell>,
 }
 
+impl<'a, F: ScalarField> From<&'a AssignedValue<F>> for AssignedValue<F> {
```

Comment:
hmm not sure about having such a blanket conversion that allows mixing between reference and non-ref. can you point to the specific use-case?

Reply:
it is needed to satisfy this trait bound
    type UnsafeFieldPoint: Clone
        + Debug
        + Send
        + Sync
        + From<Self::FieldPoint>
        + for<'a> From<&'a Self::UnsafeFieldPoint>
        + for<'a> From<&'a Self::FieldPoint>;